advocacy_docs/repos/working_with_repos/integrating/artifactory.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Using Hosted Artifactory with EDB repositories
+navTitle: Hosted Artifactory
+description: How to use Hosted JFrog Artifactory with EDB repositories
+---
+
+To set up Hosted JFrog Artifactory with EDB repositories, you need to create remote repositories in Artifactory. This article shows how to use Hosted JFrog Artifactory with EDB repositories. The process is different for on-premises Artifactory.
+
+## Get the base URLs
+
+The process is requires a base URLs to access the EDB repositories.
+
+1. From your EDB account, [get your token](../../getting_started/with_cli/get_your_token).
+
+1. Make a note of your token, for our examples, the token will be `xyz123`.
+
+1. Select the repository you want to access, for example, `standard`. :
+
+1. Your base URL will is then formed as:
+
+    ```
+    https://downloads.enterprisedb.com/<token>/<repository>/
+    ```
+
+    For the `standard` with our token, the repository, the base URL will be:
+
+    ```
+    https://downloads.enterprisedb.com/xyz123/standard/
+    ```
+
+1. Append the appropriate path for the repository type. 
+   For example, for a Red Hat Enterprise Linux 9 on x86 architecture, append `rpm/el/9/x86_64/` and seperately append `rpm/el/9/noarch/` for the noarch packages, to create two base URLs. With our example, the base URLs will be:
```

Comment:
Do we need a table that enumerates these and also mentions the Debian URLs?

Reply:
I would need the details of what is needed to form the Debian URLs in our scheme.